src/controllers/sessionController.ts

```diff
@@ -54,7 +73,7 @@ export default class SessionController {
           myPortalToken: user.myportal_token,
           spidLevel: user.spid_level,
           walletToken: user.wallet_token,
-          zendeskToken: user.zendesk_token
+          zendeskToken: user.zendesk_token + zendeskSuffix
```

Comment:
What if we use template string?
```suggestion
          zendeskToken: `${user.zendesk_token}${zendeskSuffix}`
```

Reply:
Seems legit.

Reply:
fixed in https://github.com/pagopa/io-backend/pull/926/commits/ca533a71f9f89340555f1e21f73156c352de4b6c